src/core/operators/detect_lang_of_media.py

```diff
@@ -120,6 +143,12 @@ def extract_speech(fname):
     Returns:
         str or bool: Name of the audio file with the extracted speech, False if no voice activity detected.
     """
+    extension_of_file = fname.split(".")[-1]
+    if extension_of_file != "wav":
```

Comment:
instead of doing this - shall we pass the media type as an input argument of the [run](https://github.com/tattle-made/feluda/blob/690d48a63a855cfa8675fd1099f57e8a6aacd8db/src/core/operators/detect_lang_of_audio.py#L196) function. 
- this way we have separate workflows for handling audio and video. (might make debugging easier in the future)
- I know whisper expects the audio file to be in an `.wav` format, but there is a possibility that someone might pass am `.ogg` format audio file, if that's the case the audio file will enter this if statement, which is what we want to avoid. 
- is there a specific format that the video file should be in? like `.mp4`?

wanted to know your thoughts on this

Reply:
Got it. There can be multiple extensions for video and audio format. We can change the code such that we consider all common video and audio extensions. What do you think?

Reply:
I think it will help to standardise the media format in the operator (this is also what we have been doing so far on Feluda). So for `audio` we can stick to `.wav` and for `video` to `.mp4`. 

These operators are then used as atomic functions to analyze lot of data. A [worker](https://github.com/tattle-made/feluda/blob/main/src/worker/clustering_media/clustering_media_worker.py) in Feluda is where different parts of the feluda come together for analysis of data. So say, we have to deal with media items of different format's, converting them to the standard format can be done in the `worker`. For instance, we have a function that can convert any [audio file to wav](https://github.com/tattle-made/feluda/blob/690d48a63a855cfa8675fd1099f57e8a6aacd8db/src/core/models/media_factory.py#L149-L170), so such function come in handy on the `worker`

Reply:
so this operator can assume that it will receive audio file in a `.wav` format and video file in a `.mp4` format, 
I think you still have check's for it and raise appropriate error if a media file does not the match the expected format.

Reply:
Got it. I'll make necessary changes right away.